docs/glossary.rst

```diff
@@ -94,6 +95,23 @@ email alert if something seems wrong. Only system admins connect
 to this server, and they may only do so using Tor.
 
 
+Onion Service
+-------------
+
+Tor onion services provide anonymous inbound connections to websites and other servers exclusively over the Tor network. For example, SecureDrop uses onion services for the *Journalist Interface* and *Source Interface* websites, as well as for adminsitrative access to the servers in SSH-over-Tor mode.
+
+Onion services can be accessed by clicking a link or pasting the onion service address into Tor Browser. For example, `sdolvtfhatvsysc6l34d65ymdwxcujausv7k5jk4cy5ttzhjoi6fzvyd.onion <https://sdolvtfhatvsysc6l34d65ymdwxcujausv7k5jk4cy5ttzhjoi6fzvyd.onion>`__ is SecureDrop's onion service address.
```

Comment:
We generally prefer not to link to onion services, because such links are clickable in a clearnet browser, which may cause DNS leakage before the request fails. While this is just an example address, we should follow the same convention here. Note that this example address is not reachable using `https://` (HTTPS is not required for onion services, which provide their own layer of encryption, though it can be useful as a way to assert ownership).

Reply:
I would also change "SecureDrop's onion service address" to "the onion service address for the SecureDrop website", just to be minimally ambiguous (which SecureDrop?).

Reply:
:bulb: Oh, quite a few things here, thank you for the explanations @eloquence.